app/src/main/java/com/leoart/koreanphrasebook/ui/splash/SplashActivity.kt

```diff
@@ -30,9 +33,20 @@ class SplashActivity : AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
-        model.getSyncInfo().observe(this, Observer<Boolean> {
+        model.getSyncInfo().observe(this, Observer<Resource<Boolean>> {
             it?.let {
-                onSuccessDataRefresh(it)
+                when (it.status) {
+                    Status.LOADING -> {
```

Comment:
Why state is not handled?

Reply:
@bogdanustyak there is nothing to do while loading, i added resource here just to handle possible errors